CalibTracker/SiStripQuality/plugins/SiStripQualityStatistics.cc

```diff
@@ -34,11 +34,15 @@ SiStripQualityStatistics::SiStripQualityStatistics( const edm::ParameterSet& iCo
   TkMapFileName_(iConfig.getUntrackedParameter<std::string>("TkMapFileName","")),
   fp_(iConfig.getUntrackedParameter<edm::FileInPath>("file",edm::FileInPath("CalibTracker/SiStripCommon/data/SiStripDetInfo.dat"))),
   saveTkHistoMap_(iConfig.getUntrackedParameter<bool>("SaveTkHistoMap",true)),
+  runNumberA_(iConfig.getUntrackedParameter<unsigned long long>("RunNumber",0)),
```

Comment:
@vdutta 
run numbers in CMS are an unsigned int  (0-4 million something). Why a unsigned long long?